### PR TITLE
[CI]deadlink check

### PR DESCRIPTION
**Pull request by CalvinKirs**

### CODE REVIEW

File: .github/workflows/dead-link-checker.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+concurrency:
+  group: dlc-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
```
Comment (kezhenxu94) [inline .github/workflows/dead-link-checker.yaml:31]: Not sure whether this action is in allowlist of Apache org, a safer way is to use it as a submodule

Comment (CalvinKirs) [reply .github/workflows/dead-link-checker.yaml:31]: Thanks,I will fix it

File: .dlc.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+  "ignorePatterns": [
+    {
+      "pattern": "^http://localhost"
+    },
+    {
+      "pattern": "^https://mvnrepository.com"
+    }
```
Comment (CalvinKirs) [inline .dlc.json:8]: ping mvnrepository 403～

**Review by kezhenxu94 — APPROVED** [review]
LGTM

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**